ZenPacks/zenoss/OpenStackInfrastructure/modeler/plugins/zenoss/OpenStackInfrastructure.py

```diff
@@ -1282,8 +1279,14 @@ def process(self, device, results, unused):
         quotas = []
         for quota_key in results['quotas'].keys():
             quota = results['quotas'][quota_key]
-            tenant = [t for t in quota_tenants if t['id'] == quota_key][0]
+            quota_tenant_list = [t for t in quota_tenants
```

Comment:
I think this would be clearer if rewritten as a normal loop that breaks after it finds a match, rather than building a list and using the first element in it.

Also, the warning message is a bit cryptic.. is there some action the user should take?  (I don't think so).   What are the implications of this being skipped?   Looks like a Quota component won't be created.   It appears that the quota isn't used for anything else really, so it seems harmless enough to just let that happen- i don't think a warning is required.  The quota isn't visible because its tenant is not.  That's reasonable- i don't think we even need an INFO level message about it, if this is a normal situation... just DEBUG.

Reply:
In addition, I'm changing scope of tenant to q_tenant in ZPS-4627 because there is leakage from above.